projects/packages/jetpack-mu-wpcom/src/features/wpcom-site-menu/wpcom-site-menu.php

```diff
@@ -212,6 +181,39 @@ function wpcom_add_wpcom_menu_item() {
 }
 add_action( 'admin_menu', 'wpcom_add_wpcom_menu_item' );
 
+/**
+ * Add All Sites menu to the right side of the WP logo on the masterbar.
+ */
+function add_all_sites_menu_to_masterbar() {
+	/**
+	 * Don't show `All Sites` to administrators without a WordPress.com account being attached,
+	 * as they don't have access to any of the pages.
+	 */
+	if ( ! current_user_has_wpcom_account() ) {
+		return;
+	}
+
+	global $wp_admin_bar;
```

Comment:
Super nit: this can be added as a parameter in the function (see: https://developer.wordpress.org/reference/hooks/admin_bar_menu/)